lib/models/repository-states/state.js

```diff
@@ -438,6 +449,11 @@ export default class State {
     return this.repository.state;
   }
 
+  // pipeline
+  executePipelineAction(...args) {
+    return this.repository.executePipelineAction(...args);
+  }
+
```

Comment:
Does this need to be here on `State`?

Reply:
Yeah, it's either this, or in the `Present` class we call `this.repository.executePipelineAction(...)` directly. I decided to put a method on `State` for convenient access. Open to other thoughts if you have them.